griptape/tools/base_tool.py

```diff
@@ -94,9 +94,11 @@ def schema(self) -> dict:
                 {
                     Literal("name"): self.name,
                     Literal("path", description=self.activity_description(activity)): self.activity_name(activity),
-                    Literal("input"): {"values": getattr(activity, "config")["schema"]}
-                    if self.activity_schema(activity)
-                    else {},
+                    **(
+                        {Literal("input"): {"values": getattr(activity, "config")["schema"]}}
+                        if self.activity_schema(activity)
+                        else {}
+                    ),
```

Comment:
Makes sense now but could you add a comment describing what and why is happening here? Will save someone time googling this syntax in the future.

Reply:
May be even pull it into a lambda function to make it even cleaner? Something like `**(activity_to_input(activity))` where `activity_to_input` is:

```python
activity_to_input = lambda a : {Literal("input"): {"values": getattr(a, "config")["schema"]}} if self.activity_schema(a) else {}
```